deltachat-ios/ViewModel/ContactCellViewModel.swift

```diff
@@ -32,11 +33,13 @@ struct DeaddropCellData {
     let chatId: Int
     let msgId: Int
     let summary: DcLot
+    let deaddropContact: DcContact
```

Comment:
why add and cache that here?

i'd prefer a more 1:1 adaption as in the rest of this pr.

Reply:
i added a commit that discards the additional caching, just to avoid introducing new issues as side effects here. the code is also closer to the old one and easier to review.